src/util/swapi.js

```diff
@@ -0,0 +1,23 @@
+const films = 'https://swapi.co/api/films/';
+
+async function fetchData(url) {
+    const response = await fetch(url);
```

Comment:
async/await usage is great. We don't want to see Promises and callbacks anymore 👍